### DISCUSSION TITLE
Desktop App v1.4

### DISCUSSION THREAD

**Discussion by rizwansoaib** [rizwansoaib/whatsapp-monitor#148] [General]
#144 bug fixed and improvemrnt
[Desktop-App v1.4.zip](https://github.com/rizwansoaib/whatsapp-monitor/files/8055683/Desktop-App.v1.4.zip)
 

<hr /><em>This discussion was created from the release <a href='https://github.com/rizwansoaib/whatsapp-monitor/releases/tag/Desktop%20v1.4'>Desktop App v1.4</a>.</em>

**Comment by Dewruwan95** [comment]
is there any method to work it while minimizing the app?

**Comment by maantao** [comment]
Recently downloaded and installed Windows Desktop App 1.4.0 ... but wasn't able to generate any monitoring log - ALL contacts were showing as OFFLINE, even though my second mobile number was concurrently logged in to WhatsApp.

Appreciate confirmation that the desktop app is currently working as intended.

**Comment by rizwansoaib** [comment]
I didn't updated desktop app so yes i am admitting that there are chances that is not not working.


But you can use chrome extension it's working fine as of now.

**Comment by maantao** [comment]
Thanks Rizwan for your prompt response.

Have downloaded and tried WhatsApp Monitor 2.9 Chrome Extension ... does the web.whatsapp browser window need to remain active as the foreground task in order for the online status to be tracked?

**Comment by rizwansoaib** [comment]
#145

On Fri, 26 Aug, 2022, 7:01 am maantao, ***@***.***> wrote:

> Thanks Rizwan for your prompt response.
>
> Have downloaded and tried WhatsApp Monitor 2.9 Chrome Extension ... does
> the web.whatsapp browser window need to remain active as the foreground
> task in order for the online status to be tracked?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/148#discussioncomment-3479304>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC2EZDZU726WFP3FVMOQS3V3AM5LANCNFSM5OJBN2HQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> .com>
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.

**Comment by maantao** [comment]
Thanks Rizwan for pointing to the discussion ... some salient points are still not clear, and a description of the limitations / constraints would help in understanding the way to best make use of the tool.

It is indicated that as long as web.whatsapp chrome browser session is not minimised, monitoring continues even if it is in the background. Is the purpose of pyautogui to move the mouse cursor to keep the screen / system awake and prevent sleep?

**Comment by ANKAMAM** [comment]
Thanks Rizwan. thanks for desktop 1.4 version but last wa update is app is not working firefox extention or chrome 
will you update the program ? Is there a way we can get online offline information from wa?

If the WhatsApp user has turned off the privacy in the settings, it does not come, if it is open, it comes   If the WhatsApp user has turned off the privacy in the settings, it does not come, if it is open, it comes

is there a way to hang it
